### PR TITLE
Build with cdp

### PR DESCRIPTION
**Pull request by harti2006**
Enable continuous delivery platform builds
 
- [x] stable
- [x] lts

### CODE REVIEW

**Comment by hjacobs** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
npm 3 is apparently too broken, so use something like this:

    RUN mkdir /tmp/npm_install && \
        cd /tmp/npm_install && \
        npm install npm@"$NPM_VERSION" > /dev/null && \
        /tmp/npm_install/node_modules/.bin/npm install -g npm@"$NPM_VERSION" > /dev/null && \
        /tmp/npm_install/node_modules/.bin/npm install -g npm@"$NPM_VERSION" > /dev/null && \
        rm -rf /tmp/npm_install

**Comment by harti2006** [comment]
:+1:

**Comment by harti2006** [comment]
Thanks @aermakov-zalando it works as expected now.

**Comment by aermakov-zalando** [comment]
👍

File: delivery.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+build_steps:
+    - desc: 'Install Docker'
+      cmd: |
+        # enable experimental features (--squash)
+        mkdir -p /etc/systemd/system/docker.service.d
+        cat << EOF > /etc/systemd/system/docker.service.d/docker.conf
+        [Service]
+        ExecStart=
```
Comment (ChristianLohmann) [inline delivery.yaml:8]: *gulp* i'm not familiar with CDP config yet....but i this correct? two times `ExecStart` ? This seems to be the content of the conf file but i'm wondering anyway...

Comment (harti2006) [reply delivery.yaml:8]: I copied this shamelessly from @hjacobs  😆 
E.g. here https://github.com/zalando-stups/docker-ubuntu/blob/master/delivery.yaml

Comment (ChristianLohmann) [reply delivery.yaml:8]: hehe, if it works....
👍

**Pull request merged**